### ISSUE TITLE
Questions related to Batch Norm

### ISSUE THREAD

**Issue by mrgloom** [phillipi/pix2pix#172]
As I can see here in unet like generator network 1st layer and last layer before tanh activation are without BN layer, why?
https://github.com/phillipi/pix2pix/blob/master/models.lua#L47
https://github.com/phillipi/pix2pix/blob/master/models.lua#L50
https://github.com/phillipi/pix2pix/blob/master/models.lua#L88

Also by search I found that BN was removed in the middle of encoder-decoder generator: 
https://github.com/phillipi/pix2pix/commit/b479b6b7d37f9d7e87dce7f5e627dc3bb7b4a117#commitcomment-21887022

Also in training you use batch size = 1, why?
https://github.com/phillipi/pix2pix/blob/master/train.lua#L16

I'm not familiar with torch, but is this means that at prediction time you use not train phase, i.e. not use BN layer and Dropout layer like in training phase?
https://github.com/phillipi/pix2pix/blob/master/test.lua#L23
